docs/design/datacontracts/GC.md

```diff
@@ -0,0 +1,90 @@
+# Contract GC
+
+This contract is for getting information about the garbage collector configuration and state.
+
+## APIs of contract
+
+```csharp
+public enum GCHeapType
+{
+    Unknown,
+    Workstation,
+    Server,
+}
+```
+
+```csharp
+    // Return the type of the current GC heap
+    GCHeapType GetGCHeapType();
```

Comment:
Should this rather return a string that identifies the GC type? 

The string can be a list of comma separated identifiers. We can infer workstation vs. server by looking for "workstation" and "server" identifies.

Reply:
Another identifier in the string can be `segments` vs. `regions`, or `experimental`.